ols/src/query_helpers/docs_summarizer.py

```diff
@@ -54,7 +62,7 @@ def summarize(
         query: str,
         history: Optional[str] = None,
         **kwargs: Any,
-    ) -> tuple[Response, str]:
+    ) -> tuple[Response | str, str]:
```

Comment:
we should always return the same type.  Making callers examine the object they got back to understand the type is going to be error prone.